app/controllers/v0/claims_controller.rb

```diff
@@ -9,6 +9,10 @@ def index
              each_serializer: ClaimSerializer
     end
 
+    def request_decision
+      render json: current_user.request_claim_decision(params[:id])
```

Comment:
I chose to just pass through the JSON we receive from EVSS, which is an object that looks like
```javascript
{
  success: true
}
```
Do we want to just return a 200 with no body instead on success?